libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.spec.ts

```diff
@@ -463,13 +508,78 @@ describe('Text editor', () => {
       validateToolbarActions(testComponent.toolbarActions);
     });
 
+    [
+      {
+        desc: 'new window',
+        windowOption: 'new',
+        urlStrings: [
+          'href="https://google.com',
+          'rel="noopener noreferrer"',
+          'target="_blank"',
+        ],
+        containsHintTextExpect: (
+          inputBoxText = document.querySelector('.sky-modal .sky-input-box')
+            ?.textContent
+        ) =>
+          expect(inputBoxText).toContain(
```

Comment:
Since this test imports `expect` from `@skyux-sdk/testing`, you can use the extension method `toHaveText()` which will trim whitespace and then do an exact match. This will ensure the test fails if we inadvertently add more text to this element later.

Reply:
Changed to using `toHaveText()`.